### ISSUE TITLE
Instructions are not screen-reader accessible

### ISSUE THREAD

**Issue by ekressmiller** [mplp/docassemble-MLHPPOAndProposedOrder#421]
It looks like the Instructions word documents have not been properly tagged.

But also we might need to change the assembling method because I believe I remember in the past that bundled documents lost their screen reader tags (but should ask in Slack before assuming that's what should happen). We may need to do 2 separate inserts instead.
